students/daniel_grubbs/lesson01/break_me.py

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env python3
+
+# Name error - Calls a variable that doesn't exist
+def error_name():
+    print(test_name_error)
+
+error_name()
+
+# TypeError
+def error_type():
+    a = 5
+    b = "some string"
+    return a + b
+
+
+error_type()
+
+
+# SyntaxError - left off colon
+def error_syntax()
+    print("error")
+
+
+error_syntax()
+
+# AttributeError
+def error_attr(a, b):
+    import collections
+    test = collections.noWayThisExists(a, b)
+    return test
+
+
+error_attr(5, 8)
+
+import collections
```

Comment:
imports for the file should be done at the top of the file, but we'll get to that.  Your import inside the `def` though is allowed although frowned upon unless you are protecting against circular imports.  If none of that makes sense don't worry about it.  That will be for the advanced class ;)